docs/references/architecture/recipes/bridge.md

```diff
@@ -52,8 +52,9 @@ The Bridge recipe is a secure pattern where messages are passed between brokers
 
 import { colors } from '@theme/Mermaid'
 
-```mermaid
-graph TD
+<!-- prettier-ignore-start -->
+
+<Mermaid chart={`graph TD
```

Comment:
Across all of these, we prefer to keep the old way as it allows the graph to be viewed outside of the rendered site (such as on GitHub). You can see an example of this here: https://github.com/tauri-apps/tauri-docs/tree/dev/docs/references/architecture/inter-process-communication

Reply:
Ah, yes, it is rendered in GitHub; I did not see that.
The issue is that it is not rendered by Docusaurus... (when using `yarn && yarn start`, the recipes pages are rendered without the diagrams) :disappointed: 

For the `<Mermaid chart={...} />` syntax, I took it from https://github.com/tauri-apps/tauri-docs/blob/f743e35c3436d953485c5a8932d93d957c0def51/docs/references/architecture/readme.md?plain=1.

Reply:
Oh, but actually it works for `references/architecture/inter-process-communication/` so of course it should also work here. I guess there's another issue. I'll check it.

Reply:
I just checked, and I think the issue is caused by the `style` lines. If I remove them then they render correctly. I assume they only work if they're in a `<Mermaid />` component (which is probably why Architecture/readme has that instead of the code block format).

But regardless, those docs were created a while ago which is why they are formatted like that, the preferred way would be to use code block syntax so that we get that nice GitHub rendering.

Reply:
I reverted to the previous syntax, and used classes to style the elements. It is rendered with the proper styles with Docusaurus, and degrades gracefully on GitHub.